perceptia/perceptiactl/perceptiactl.rs

```diff
@@ -49,8 +55,13 @@ fn main() {
         ("about", Some(_)) => {
             about::process();
         }
-        ("screenshot", Some(_)) => {
-            screenshot::process();
+        ("screenshot", Some(subcommand)) => {
+            if let Some(path) = subcommand.value_of("path") {
+                screenshot::process(String::from(path));
+            } else {
+                let now = chrono::Local::now();
+                screenshot::process(now.format("%Y-%m-%d_%H%M%S").to_string());
```

Comment:
This file name will not work. It has to have extension. Without it `image` crate fails to determine the format. What do you think about adding some prefix? "screenshot-%Y-%m-%d_%H%M%S.png" would be good.